apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTableFixture.cs

```diff
@@ -30,9 +31,15 @@ public SpannerTableFixture(string tableName)
             TableName = tableName;
             if (Database.Fresh)
             {
+                Logger.DefaultLogger.Debug($"Creating table {TableName}");
```

Comment:
No possibility of retries for CreateTable?

Reply:
I'd expect that to be DDL, which doesn't suffer the same sort of aborting issues as data modifications. (I don't think I've ever seen it fail there, anyway.)

Reply:
OK